gui/src/components/WagmiWrapper.tsx

```diff
@@ -36,7 +37,7 @@ export function WagmiWrapper({ children }: Props) {
     setClient(buildClient(network));
   }, [network]);
 
-  if (!client) return <></>;
+  if (!client) return null;
```

Comment:
is this allowed? I remember seeing an error at some point when doing this

Reply:
Yap, I think so, this is what I've always done. I also didn't see any error.